conda-{activate, clean, compare}: add page. conda-create: add prefix … (#18051)
tldr-pages/tldr: pages/common/conda-compare.md

# conda compare

> Compare packages between conda environments.
> More information: <https://docs.conda.io/projects/conda/en/stable/commands/compare.html>.

- Compare packages in the current directory to packages from the file `file.yml`:

`conda compare file.yml`

> - Compare packages in environment [n]amed `myenv` to packages from the file `file.yml`:

Reviewer: ```suggestion
- Compare packages in environment named `myenv` to packages from the file `file.yml`:
```

> `conda compare {{[-n|--name]}} myenv {{/path/to/file.yml}}`

Reviewer: ```suggestion
`conda compare {{[-n|--name]}} myenv {{path/to/file.yml}}`
```

> - Compare packages in environment `myenv` at custom [p]ath (i.e. [p]refix) to packages from the file `file.yml`:

Reviewer: ```suggestion
- Compare packages in environment `myenv` at custom path (i.e. prefix) to packages from the file `file.yml`:
```

> `conda compare {{[-p|--prefix]}} {{/path/to/myenv}} {{/path/to/file.yml}}`

Reviewer: ```suggestion
`conda compare {{[-p|--prefix]}} {{path/to/myenv}} {{path/to/file.yml}}`
```

> - Display detailed [h]elp message:

Reviewer: ```suggestion
- Display help:
```
